docker/tests/scripts/test-fedora-binaries.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+# SPDX-FileCopyrightText: 2023 Oxhead Alpha
+# SPDX-License-Identifier: LicenseRef-MIT-OA
+
+set -e
+
+for version in $(jq -r '.fedora[]' docker/supported_versions.json); do
+    docker build --build-arg dist="$version" -t fedora-test -f docker/tests/Dockerfile-fedora-test .
+    docker run -i fedora-test
```

Comment:
Could you elaborate why `-i` is needed? AFAIR, it forwards stdin, but I assume that it's not needed here